pkg/kubelet/cri/remote/remote_image.go

```diff
@@ -99,7 +99,7 @@ func (r *remoteImageService) determineAPIVersion(conn *grpc.ClientConn) error {
 		r.imageClientV1alpha2 = runtimeapiV1alpha2.NewImageServiceClient(conn)
 
 	} else {
-		return fmt.Errorf("unable to determine image API version: %w", err)
+		return fmt.Errorf("unable to determine image API version, maybe your image not support CRI: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("unable to determine image API version or API is not implemented: %w", err)
```

Reply:
it may be a good idea to incldue the socket address into this message as well

Reply:
OK, updated.
PTAL thanks.